trae_api/api/agent/endpoints/views.py

```
@@ -0,0 +1,606 @@
+        request_logger.info(
+            "Streaming response created successfully",
+            format_type=format_type,
+            media_type=media_type,
+            setup_duration_ms=duration_ms,
+        )
+        
+        return response
+        
+    except Exception as e:
+        duration_ms = (time.time() - request_start_time) * 1000
+        log_request_complete(request_logger, 500, None, duration_ms)
```

Ruff (0.12.2)

Undefined name `log_request_complete`

(F821)